src/displayapp/DisplayApp.cpp

```diff
@@ -129,15 +129,6 @@ void DisplayApp::Start(System::BootErrors error) {
 
   bootError = error;
 
-  lvgl.Init();
-  motorController.Init();
```

Comment:
just to be sure, SystemTask doesn't need lvgl and motorcontroller initialized directly after the `Start()` call, does it?

Reply:
I assume SystemTask is calling `DisplayApp.Start()` haven't actually checked

Reply:
Motor and LVGL are both "owned" by DisplayApp (no other task ever runs any functions from them) so it should be OK to defer them